packages/@icon-magic/cli/src/index.ts

```diff
@@ -54,46 +59,70 @@ program
   });
 
 program
-  .command(
-    'distribute',
+  .command('distribute [inputPaths...]')
```

Comment:
Consider using angle brackets to denote required arguments.

Reply:
Angle brackets don't ... for all arguments